python313Packages.python-whois: 0.9.5 -> 0.9.6

fabaff opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff committed:
python313Packages.python-whois: 0.9.5 -> 0.9.6

thiagokokada commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 452894`
Commit: [`eebcf391bfb21ee19179758a8c2391a1ad20649b`](https://github.com/NixOS/nixpkgs/commit/eebcf391bfb21ee19179758a8c2391a1ad20649b) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/eebcf391bfb21ee19179758a8c2391a1ad20649b..pull/452894/head))
Merge: [`48700a6ec84dd28245dd9fbb67c66171c9c70132`](https://github.com/NixOS/nixpkgs/commit/48700a6ec84dd28245dd9fbb67c66171c9c70132)

Logs: https://github.com/thiagokokada/nixpkgs-review-gha/actions/runs/18593460059


---
### `x86_64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>baddns</li>
    <li>baddns.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 7 packages built:</summary>
  <ul>
    <li>offensive-azure</li>
    <li>offensive-azure.dist</li>
    <li>python312Packages.python-whois</li>
    <li>python312Packages.python-whois.dist</li>
    <li>python313Packages.python-whois</li>
    <li>python313Packages.python-whois.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>baddns</summary>
<pre>        else:
            registered_domain = ext.registered_domain
        log.debug(f&quot;Extracted base domain [{registered_domain}] from [{self.target}]&quot;)
        log.debug(f&quot;Submitting WHOIS query for {registered_domain}&quot;)
        try:
            w = await asyncio.to_thread(whois.whois, registered_domain, quiet=True)
            log.debug(f&quot;Got response to whois request for {registered_domain}&quot;)
            self.whois_result = {&quot;type&quot;: &quot;response&quot;, &quot;data&quot;: w}
&gt;       except whois.parser.PywhoisError as e:
               ^^^^^^^^^^^^^^^^^^^^^^^^^
E       AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;

/build/source/baddns/lib/whoismanager.py:28: AttributeError
=========================== short test summary info ============================
FAILED tests/cli_test.py::test_cli_cname_nxdomain - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_js - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_css - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_csp - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_cors - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
================= 5 failed, 30 passed, 10 deselected in 0.95s ==================</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>baddns</li>
    <li>baddns.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 7 packages built:</summary>
  <ul>
    <li>offensive-azure</li>
    <li>offensive-azure.dist</li>
    <li>python312Packages.python-whois</li>
    <li>python312Packages.python-whois.dist</li>
    <li>python313Packages.python-whois</li>
    <li>python313Packages.python-whois.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>baddns</summary>
<pre>        else:
            registered_domain = ext.registered_domain
        log.debug(f&quot;Extracted base domain [{registered_domain}] from [{self.target}]&quot;)
        log.debug(f&quot;Submitting WHOIS query for {registered_domain}&quot;)
        try:
            w = await asyncio.to_thread(whois.whois, registered_domain, quiet=True)
            log.debug(f&quot;Got response to whois request for {registered_domain}&quot;)
            self.whois_result = {&quot;type&quot;: &quot;response&quot;, &quot;data&quot;: w}
&gt;       except whois.parser.PywhoisError as e:
               ^^^^^^^^^^^^^^^^^^^^^^^^^
E       AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;

/build/source/baddns/lib/whoismanager.py:28: AttributeError
=========================== short test summary info ============================
FAILED tests/cli_test.py::test_cli_cname_nxdomain - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_js - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_css - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_csp - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_cors - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
================= 5 failed, 30 passed, 10 deselected in 0.77s ==================</pre>
</details>
</details>

---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>baddns</li>
    <li>baddns.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 7 packages built:</summary>
  <ul>
    <li>offensive-azure</li>
    <li>offensive-azure.dist</li>
    <li>python312Packages.python-whois</li>
    <li>python312Packages.python-whois.dist</li>
    <li>python313Packages.python-whois</li>
    <li>python313Packages.python-whois.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>baddns</summary>
<pre>        else:
            registered_domain = ext.registered_domain
        log.debug(f&quot;Extracted base domain [{registered_domain}] from [{self.target}]&quot;)
        log.debug(f&quot;Submitting WHOIS query for {registered_domain}&quot;)
        try:
            w = await asyncio.to_thread(whois.whois, registered_domain, quiet=True)
            log.debug(f&quot;Got response to whois request for {registered_domain}&quot;)
            self.whois_result = {&quot;type&quot;: &quot;response&quot;, &quot;data&quot;: w}
&gt;       except whois.parser.PywhoisError as e:
               ^^^^^^^^^^^^^^^^^^^^^^^^^
E       AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;

/nix/build/nix-5769-2378733609/source/baddns/lib/whoismanager.py:28: AttributeError
=========================== short test summary info ============================
FAILED tests/cli_test.py::test_cli_cname_nxdomain - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_js - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_css - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_csp - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_cors - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
================= 5 failed, 30 passed, 10 deselected in 1.15s ==================</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>baddns</li>
    <li>baddns.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 7 packages built:</summary>
  <ul>
    <li>offensive-azure</li>
    <li>offensive-azure.dist</li>
    <li>python312Packages.python-whois</li>
    <li>python312Packages.python-whois.dist</li>
    <li>python313Packages.python-whois</li>
    <li>python313Packages.python-whois.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>baddns</summary>
<pre>        else:
            registered_domain = ext.registered_domain
        log.debug(f&quot;Extracted base domain [{registered_domain}] from [{self.target}]&quot;)
        log.debug(f&quot;Submitting WHOIS query for {registered_domain}&quot;)
        try:
            w = await asyncio.to_thread(whois.whois, registered_domain, quiet=True)
            log.debug(f&quot;Got response to whois request for {registered_domain}&quot;)
            self.whois_result = {&quot;type&quot;: &quot;response&quot;, &quot;data&quot;: w}
&gt;       except whois.parser.PywhoisError as e:
               ^^^^^^^^^^^^^^^^^^^^^^^^^
E       AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;

/nix/build/nix-6493-3262421914/source/baddns/lib/whoismanager.py:28: AttributeError
=========================== short test summary info ============================
FAILED tests/cli_test.py::test_cli_cname_nxdomain - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_js - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_css - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_csp - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
FAILED tests/references_test.py::test_references_direct_cors - AttributeError: module &#x27;whois.parser&#x27; has no attribute &#x27;PywhoisError&#x27;
================= 5 failed, 30 passed, 10 deselected in 0.56s ==================</pre>
</details>
</details>

fabaff committed:
baddns: 1.11.236 -> 1.12.294

Diff: https://github.com/blacklanternsecurity/baddns/compare/1.11.236...1.12.294

Changelog: https://github.com/blacklanternsecurity/baddns/releases/tag/1.12.294
